app/services/support_interface/qualifications_export.rb

```diff
@@ -5,7 +5,9 @@ def data_for_export
                                 .select(:id, :application_form_id, :rejection_reason, :structured_rejection_reasons, :status, :course_option_id)
                                 .includes(:course_option, :course, :provider)
 
-      application_choices.map do |application_choice|
+      data_for_export = []
+
+      application_choices.find_each(batch_size: 100) do |application_choice|
```

Comment:
I think you can use `find_each(batch_size: 100).lazy.map` if you want to avoid the array concatenation.

Reply:
💯 Yep that works!